### ISSUE TITLE
Invite triggers should use entire phrases with spaces, like other word lists

### ISSUE THREAD

**Issue by fuser55** [iraizo/AutoLayer#29]
If I use an invite trigger like "lf layer", it will try inviting a player that posts "lf1m" or "lf dps" instead of using the entire phrase for the trigger. This is a pretty large deviation of behavior IMO since the same tooltip appears for all 3 word lists, and it makes so that you can't specify invite trigger phrases.

Multi-word invite triggers are important to avoid accidental invites, for example if someone is talking about layering in a chat but not specifically looking for a layer invite.

https://github.com/iraizo/AutoLayer/blob/1ca9f293a156e96582a1051af901d7c8fb90cca1/configuration.lua#L33

It appears that the trigger parsing doesn't use the same regex.

https://github.com/iraizo/AutoLayer/blob/1ca9f293a156e96582a1051af901d7c8fb90cca1/configuration.lua#L50

**Comment by fuser55** [comment]
I also don't think that "layer" is a great default trigger. It is encouraging players to just say "layer" in LFG when using this in guild chat when not actually looking for layers can be disruptive.

**Comment by iraizo** [comment]
I have to look into this later, this code has been rewritten by a contributor.

**Comment by BannZay** [comment]
After using addon for a while, I noticed that I am getting messages from people confused why did I send them an invite. The trigger has many false positives.

Also, another idea would be to let player decide what channels they want addon to scan. IMO, addon should only scan "layer" channel, and the others channels (like guild, General, etc) should be optional.

**Comment by iraizo** [comment]
I agree with you @BannZay , filtering between channels should also be a thing but that should be another issue 😉

**Comment by ronniedude** [comment]
@iraizo Since PR #49  has been merged this issue should be closed complete
